source/_components/switch.telnet.markdown

```diff
@@ -39,6 +39,6 @@ Configuration variables:
     - **port** (*Optional*): Port to connect to. Default is 23 if not defined.
     - **command_on** (*Required*): Command to turn device on.
     - **command_off** (*Required*): Command to turn device off.
-    - **command_state** (*Required*): Command to determine the state of the switch. If not defined the switch will assume successful state changes.
-    - **value_template** (*Required*): The template evaluating to `true` will indicate that the switch is on.
+    - **command_state** (*Optional*): Command to determine the state of the switch. If not defined the switch will assume successful state changes.
+    - **value_template** (*Optional*): The template evaluating to `true` will indicate that the switch is on.
```

Comment:
`value_template` doesn't see to be [optional](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/switch/telnet.py#L28).

Reply:
Oh, you want to change it.